error: unexpected token, expected one of: "fs", "network", "operation", or "local_cells"
 --> tests/function/fail_attribute_invalid_args.rs:9:25
  |
9 | #[turbo_tasks::function(invalid_argument)]
  |                         ^^^^^^^^^^^^^^^^
